dependencies/README.md

```diff
@@ -88,3 +90,90 @@ The tool allows users to choose the scope of entry point classes:
   - **All classes in the input class path**: when reachability check is off, then
     all static linkage errors from all classes in the classpath, regardless of the reachability,
     are reported.
+
+<a name="maven_dependency_graph"></a>
+# Maven Dependency Graph
+
+A Maven dependency graph is a graph data structure where
+- Node: a node is a Maven artifact identified by [Maven coordinates][1] such as
+  `com.example.foo:bar:1.5.4`. A node may be marked _unavailable_ when the artifact is not available
+  in Maven repositories.
+- Edge: a directed edge is a dependency from one Maven artifact (_source_ of the edge)
+  to another Maven artifact (_target_ of the edge).
+
+  A dependency has a boolean attribute `optional` and an enum attribute `scope`,
+  among other properties listed in [POM Reference: Dependencies][2].
+
+  Self-loops are not possible. A parallel edge is allowed but is dropped in the model.
+
+## Comparison with Maven Dependency Trees by pom.xml
+
+Compared to a _Maven dependency tree_ retrieved by a pom.xml through
+`RepositorySystem.resolveDependencies`, a Maven dependency graph has the following differences:
+
+- A Maven dependency tree cannot have multiple versions of a Maven artifact, whereas a Maven
```

Comment:
So you're assuming this is a post-mediation tree as opposed to a pre-mediation tree?